### DISCUSSION TITLE
Download sensor deployment Guide through falcon API via Python script.

### DISCUSSION THREAD

**Discussion by afroz1** [CrowdStrike/falconpy#837] [Q&A] [answered]
Hi,

I need your help to write a python script to download the deployment guide through falcon API.

https://falcon.eu-1.crowdstrike.com/documentation/

We have to download the following Guide manually every 2-3 months in our local storage. I would like to automate things and will schedule the script every month to get the updated deployment guide every month.

Sensor Deployment:
----------------------
[Falcon Sensor for Windows](https://falcon.eu-1.crowdstrike.com/documentation/23/falcon-sensor-for-windows)
[Falcon Sensor for Mac](https://falcon.eu-1.crowdstrike.com/documentation/22/falcon-sensor-for-mac)
[Falcon Sensor for Linux](https://falcon.eu-1.crowdstrike.com/documentation/20/falcon-sensor-for-linux)
[Falcon Container Sensor for Linux](https://falcon.eu-1.crowdstrike.com/documentation/146/falcon-container-sensor-for-linux)

Sensor Deployment and Maintenance:
------------------------------------------
[Cloud IP Addresses](https://falcon.eu-1.crowdstrike.com/documentation/65/cloud-ip-addresses)

[Linux Supported Kernels](https://falcon.eu-1.crowdstrike.com/documentation/245/linux-supported-kernels)

**Comment by jshcodes** [comment, marked as answer]
Hi @afroz1!

Currently FalconPy cannot perform this operation as there is not a public API exposed to do so.  There are a couple of ideas posted by other customers related to this ask.  I'd encourage you to check them out, comment on your scenario, and then vote for what you'd like to see. This way the product team knows and has the opportunity to put it on the roadmap.

- https://eu-1.ideas.crowdstrike.com/ideas/IDEA-I-7262
- https://eu-1.ideas.crowdstrike.com/ideas/IDEA-I-8578

Thank you for the question! 😄
